packages/rocketchat-tokenpass/server/functions/lendTokenpassToken.js

```diff
@@ -0,0 +1,43 @@
+import moment from 'moment';
+
+let userAgent = 'Meteor';
+if (Meteor.release) { userAgent += `/${ Meteor.release }`; }
+
+RocketChat.lendTokenpassToken = function(lending) {
+	const authApiToken = RocketChat.settings.get('Accounts_OAuth_Tokenpass_id');
+	const authApiSecret = RocketChat.settings.get('Accounts_OAuth_Tokenpass_secret');
+	const authApiNonce = moment().unix();
```

Comment:
not sure this works as expected. [looks like](http://momentjs.com/docs/#/parsing/unix-timestamp-milliseconds/) you need to pass a timestamp to it..

but anyways, if you need a current timestamp, I would suggest changing it to `Date.now()`

Reply:
It's works without I to pass a timestamp args, since a "moment()" already contain a current timestamp... But, I did your suggestion change because it's better (more pure, what I means). Thanks for your review :-)